packages/loading-indicator/src/Spinner/Spinner.styles.ts

```diff
@@ -0,0 +1,39 @@
+import { css } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+import { Variant } from './Spinner.types';
+
+// Constants
+
+export const SpinnerSizes: Record<Variant, number> = {
+  default: 24,
+  large: 48,
+  xlarge: 64,
+  horizontal: 24,
+};
+
+export const SpinnerBottomMargins: Record<Variant, number> = {
+  default: spacing[2],
+  large: spacing[2],
+  xlarge: spacing[3],
+  horizontal: 0,
+};
+
+// Styles
```

Comment:
nit but not intuitive to have a header be equidistant to content on both sides. Should be more closely aligned to what it's describing than to the previous section.